fedora/bugzilla.py

```diff
@@ -19,5 +19,5 @@ async def bug(self, evt: MessageEvent, bug_id: str) -> None:
         await evt.mark_read()
         result = await self.bugzillaclient.get_bug(bug_id)
         await evt.respond(
-            f"[RHBZ#{bug_id}](https://bugzilla.redhat.com/{bug_id}): {result['bugs'][0]['summary']}"
+            f"[RHBZ#{bug_id}](https://bugzilla.redhat.com/{bug_id}): [{result['bugs'][0]['component'][0]}]: {result['bugs'][0]['summary']}"
```

Comment:
Not sure how to break up this line to satisfy the linter.

Reply:
you just need to break it up like we do here:

https://github.com/fedora-infra/maubot-fedora/blob/main/fedora/cookie.py#L134-L138